libs/common/src/helpers/is-gef-facility-type.test.ts

```diff
@@ -0,0 +1,19 @@
+import { isGefFacilityType } from './is-gef-facility-type';
+import { FACILITY_TYPE } from '../constants';
+
+describe('is-gef-facility-type helper', () => {
+  it('should return true when type is `CASH`', () => {
```

Comment:
could CASH in the it statement come from the constants please?

Reply:
Refactored, this file is no longer changed from before